### DISCUSSION TITLE
need help with two variables in config.xml

### DISCUSSION THREAD

**Discussion by ianqsong** [d2phap/ImageGlass#696] [Q&A] [answered]
### System information:
 - Windows OS version: 10
 - ImageGlass version: 7.5

### Questions
It seems "FrmMainWindowsBound" is about initial position of the frame, but I can't find any explanation on its value, like "749;365;981;602", especially the last three numbers ( I guess the first one among the four numbers is the left coordinate of the rectangular window.)

Also, what is the meaning of "FrmSettingsWindowsBound"?

Many Thanks.

**Comment by d2phap** [comment, marked as answer]
Hi @ianqsong 

`Bound` contains 4 numbers to define a [rectangle](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.rectangle?view=netframework-4.8) of the window: `X;Y;Width;Height`
- `X` and `Y` are coordinates

https://github.com/d2phap/ImageGlass/blob/master/Source/ImageGlass/frmMain.cs#L2651

**Comment by ianqsong** [comment]
I see. Thanks again.
What about the 4 numbers of "FrmSettingsWindowsBound"?

**Comment by d2phap** [comment]
> What about the 4 numbers of "FrmSettingsWindowsBound"?

It's same!

**Comment by ianqsong** [comment]
Actually I was wondering which frame the 4 numbers of "FrmSettingsWindowsBound" are setting -- I guess not the initial frame when ig starts. Thanks.

**Comment by d2phap** [comment]
> Actually I was wondering which frame the 4 numbers of "FrmSettingsWindowsBound" are setting -- I guess not the initial frame when ig starts. Thanks.

Oh it's the Settings dialog (Ctrl+Shift+P)
